lib/resources/campaign.js

```diff
@@ -156,6 +156,20 @@ export class LinkUnlinkNumberResponse {
         });
     }
 
+         /**
+     * import Campaign
+     * @method
+     * @param {string} campaign_id  
+     * @param {string} campaign_alias  
+     * @promise {object} return {@link PlivoGenericResponse} object
+     * @fail {Error} return Error
+     */
+     importcamp(campaign_id, campaign_alias, params = {}) {
```

Comment:
can we have better function name? please keep the function similar across the SDK